horace_core/sqw/@accumulate_headers_job/private/read_input_headers_.m

```diff
@@ -40,13 +40,15 @@
         det_tmp.filename = det.filename;
         det_tmp.filepath = det.filepath;
     end
+%{
     if ~equal_to_tol(det,det_tmp,[1.e-9,1.e-9])
         % CM:we will need to get rid of this once detpars start becoming
         % different between runs
         error('HORACE:write_nsqw_to_sqw:invalid_argument',...
             ' Detector parameters are not the same in first and %d file %s',...
             i,infiles{i});
     end
+    %}
```

Comment:
If we're removing this (as per comment) probably wants properly removing

Reply:
yes